addons/sourcemod/configs/vsh/vsh.cfg

```diff
@@ -1334,7 +1334,9 @@
 		
 		"327"	//Claidheamh Mòr
 		{
-			"desp"			"Claidheamh Mòr: {positive}Refills 25% of your charge meter on hit"
+			"desp"			"Claidheamh Mòr: {positive}Refills 25% of your charge meter on hit, {neutral}replaces damage vulnerability with slower attack speed"
+			"attrib"		"5 ; 1.15"	
```

Comment:
`412 ; 1.0` need to be readded so replaces damage vulnerability actually comes into effect

Reply:
added